src/main/resources/cassandratablecolumn.properties

```diff
@@ -43,4 +43,12 @@ filename=fileName
 rootorgid=rootOrgId
 filepath=filePath
 mapid=mapId
-sborgid=sbOrgId
\ No newline at end of file
+sborgid=sbOrgId
+contexttype=contextType
+contexttypeid=contextTypeId
+acquiredchannel=acquiredChannel
+additionalparams=additionalParams
+requestedby=requestedBy
+contextstatus=contextStatus
+createdat=createdAt
+updatedat=updatedAt
```

Comment:
can we use createdDate and updatedDate - existing values instead of new ones?
since this is used only in backend - no need to change the API details ??